doc/modules/linear_model.rst

```diff
@@ -897,6 +897,119 @@ to warm-starting (see :term:`Glossary <warm_start>`).
     .. [9] `"Performance Evaluation of Lbfgs vs other solvers"
             <http://www.fuzihao.org/blog/2016/01/16/Comparison-of-Gradient-Descent-Stochastic-Gradient-Descent-and-L-BFGS/>`_
 
+.. _Generalized_linear_regression:
+
+Generalized Linear Regression
+=============================
+
+Generalized Linear Models (GLM) extend linear models in two ways
+[10]_. First, the predicted values :math:`\hat{y}` are linked to a linear
+combination of the input variables :math:`X` via an inverse link function
+:math:`h` as
+
+.. math::    \hat{y}(w, X) = h(Xw).
```

Comment:
`Xw` could be confused with a single variable Xw, and I would rather write the `\cdot` explicitly between them (here and below).

Reply:
The UG of linear_models uses Xw everywhere (see [e.g. here](https://scikit-learn.org/dev/modules/linear_model.html#ridge-regression)). I have no strong opinion regarding Xw, but we should be consistent with the rest.

Also note that the previous version was treating X_i both as a feature *and* as sample

Reply:
> The UG of linear_models uses Xw everywhere (see e.g. here).

Fair enough.